tls/s2n_client_hello.c

```diff
@@ -202,8 +203,11 @@ static int s2n_parse_client_hello(struct s2n_connection *conn)
     GUARD(s2n_stuffer_read_uint8(in, &num_compression_methods));
     GUARD(s2n_stuffer_skip_read(in, num_compression_methods));
 
+    const struct s2n_ecc_preferences *ecc_pref = conn->config->ecc_preferences;
```

Comment:
Add the notnull_check for conn->config here, it doesn't look like it is done previously in this function.

Reply:
Will do a scan for the `notnull_check` again and will add in places I missed. Thanks for catching this.